src/main/java/strategy/HandlersStore.java

```diff
@@ -0,0 +1,17 @@
+package strategy;
+
+import java.util.Map;
+import model.FruitTransaction;
+import strategy.handler.Handler;
+
+public class HandlersStore {
```

Comment:
let's remove this class and pass Map inside of StrategyImplementation constructor